python/straw/straw.py

```diff
@@ -315,65 +350,64 @@ def readBlock(req, size):
+        binYOffset = struct.unpack('<i', uncompressedBytes[8:12])[0]
+        useShort = struct.unpack('<b', uncompressedBytes[12:13])[0]
+        type_ = struct.unpack('<b', uncompressedBytes[13:14])[0]
+        index = 0
+        if (type_ == 1):
+            rowCount = struct.unpack('<h', uncompressedBytes[14:16])[0]
             temp = 16
             for i in range(rowCount):
-                y = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                temp=temp+2
+                y = struct.unpack('<h', uncompressedBytes[temp:(temp + 2)])[0]
+                temp = temp + 2
                 binY = y + binYOffset
-                colCount = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                temp=temp+2
+                colCount = struct.unpack('<h', uncompressedBytes[temp:(temp + 2)])[0]
+                temp = temp + 2
                 for j in range(colCount):
-                    x = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                    temp=temp+2
+                    x = struct.unpack('<h', uncompressedBytes[temp:(temp + 2)])[0]
+                    temp = temp + 2
                     binX = binXOffset + x
-                    if (useShort==0):
-                        c = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                        temp=temp+2
+                    if (useShort == 0):
+                        c = struct.unpack('<h', uncompressedBytes[temp:(temp + 2)])[0]
+                        temp = temp + 2
                         counts = c
                     else:
-                        counts = struct.unpack('<f',uncompressedBytes[temp:(temp+4)])[0]
-                        temp=temp+4
+                        counts = struct.unpack('<f', uncompressedBytes[temp:(temp + 4)])[0]
+                        temp = temp + 4
                     record = dict()
                     record['binX'] = binX
                     record['binY'] = binY
                     record['counts'] = counts
                     v.append(record)
                     index = index + 1
-        elif (type_== 2):
-            temp=14
-            nPts = struct.unpack('<i',uncompressedBytes[temp:(temp+4)])[0]
-            temp=temp+4
-            w = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-            temp=temp+2
+        elif (type_ == 2):
+            temp = 14
+            nPts = struct.unpack('<i', uncompressedBytes[temp:(temp + 4)])[0]
+            temp = temp + 4
+            w = struct.unpack('<h', uncompressedBytes[temp:(temp + 2)])[0]
+            temp = temp + 2
             for i in range(nPts):
-                row=int(i/w)
-                col=i-row*w
-                bin1=int(binXOffset+col)
-                bin2=int(binYOffset+row)
-                if (useShort==0):
-                    c = struct.unpack('<h',uncompressedBytes[temp:(temp+2)])[0]
-                    temp=temp+2
+                row = int(i / w)
```

Comment:
`@sa501428, your editor adds too many edits due to styling. First, this styling doesn't follow PEP's (https://www.python.org/dev/peps/pep-0008/). Second, it changes the history of `git blame` and will make a debugger life harder as well as the reviewers for this restructuring commit.

Reply:
This should be resolved now.